skyportal/handlers/api/gcn.py

```diff
@@ -2216,3 +2220,95 @@ async def get(self, dateobs, localization_name):
                         os.remove(f)
                     except:  # noqa E722
                         pass
+
+
+class LocalizationCrossmatchHandler(BaseHandler):
+    @auth_or_token
+    async def get(self):
+        """
+        ---
+        description: Download a GCN localization skymap
+        tags:
+          - localizations
+        parameters:
+          - in: path
+            name: dateobs
+            required: true
+            schema:
+              type: dateobs
+          - in: path
+            name: localization_name
+            required: true
+            schema:
+              type: localization_name
+        responses:
```

Comment:
I think this should be something like:

        responses:
          200:
            description: A fits file corresponding to the intersection of the input fits files.
            content:
              application/fits:
                schema:
                  type: string
                  format: binary